### PR TITLE
GLOB-22920: Move to Circle 2.0

### PR DESCRIPTION
**Pull request by moshez**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,13 +20,13 @@
         "pyahocorasick>=1.1.6",
         "pycountry>=17.9.23",
         "unidecode>=1.0.22",
+        "parameterized>=0.6.1",
```
Comment (piercefreeman) [inline setup.py:23]: Why are we switching `parameterized` to a regular dependency?

Comment (moshez) [reply setup.py:23]: Because our current infrastructure doesn't handle test-only dependencies well :(

Comment (piercefreeman) [reply setup.py:23]: Interesting, so this is the case across the board?

**Review by piercefreeman — APPROVED** [review]
LGTM

File: setup.py (hunk 1 above)
Comment (moshez) [reply setup.py:23]: Hopefully I'm going to fix it soon, and then we can move it back :)
You'll be surprised how few test-only deps we have.

**Pull request merged**